src/components/LoginForm/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { reduxForm, Field } from 'redux-form/immutable';
+import PropTypes from 'prop-types';
```

Comment:
It is good practice to just import the propTypes that you need so instead of `PropTypes` you should use `{ bool, func, string }`. Use this in the other components as well.